manifests/ca.pp

```diff
@@ -157,59 +174,65 @@
       ca_hostname                    => $facts['fqdn'],
       ca_port                        => $https_port,
       admin_user                     => $admin_user,
+      service_timeout                => $service_timeout,
+      kra_config                     => $kra_config,
       package_ensure                 => $package_ensure,
       require                        => Simp_pki_service::Ca::Service[$name]
     }
   }
 
-  if $enable_scep {
-    simp_pki_service::dogtag::config_item { "Enable SCEP on ${name}":
-      ca_id   => $name,
-      key     => 'ca.scep.enable',
-      value   => true,
-      require => Exec["Configure SIMP ${name} CA"]
+  if $ca_config['ca.scep.enable'] == true {
+    # Updating auths.instance.flatFileAuth.deferOnFailure is required by
```

Comment:
Updating => Disabling ?  Wording isn't clear.  Also, why is this required? Is it problematic if the user overrides?